src/Microsoft.HttpRepl/Resources/Strings.resx

```diff
@@ -386,4 +386,16 @@ When +history option is specified, commands specified in the text file will be a
   <data name="Options" xml:space="preserve">
     <value>Options:</value>
   </data>
+  <data name="BaseHttpCommand_Error_ContentFileDoesNotExist" xml:space="preserve">
+    <value>The specified content file, "{0}", does not exist.</value>
+    <comment>{0} is the user supplied content file path</comment>
+  </data>
+  <data name="BaseHttpCommand_Error_DefaultEditorDoesNotExist" xml:space="preserve">
+    <value>The specified default editor, "{0}", does not exist.</value>
```

Comment:
Personal preference, but I think it'd be more clear if you added "The specified default editor path,..." to make it obvious that it's a file path issue.